source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/ChargeLinks/Handlers/IChargeLinkEventPublishHandler.cs

```diff
@@ -19,6 +19,6 @@ namespace GreenEnergyHub.Charges.Application.ChargeLinks.Handlers
 {
     public interface IChargeLinkEventPublishHandler
     {
-        Task HandleAsync(ChargeLinkCommandAcceptedEvent command);
+        Task HandleAsync(ChargeLinkCommandAcceptedEvent acceptedEvent);
```

Comment:
As we don't like `RequestReply` naming this should probably also be:
```suggestion
        Task HandleAsync(ChargeLinkAcceptedEvent acceptedEvent);
```
But feel free to scope this out of this PR.